Library/Formula/task.rb

```diff
@@ -14,6 +14,8 @@ class Task < Formula
 
   depends_on "cmake" => :build
   depends_on "gnutls" => :optional
+	# GnuTLS is not optional any more starting with 2.5.0
```

Comment:
The indent is wrong here. Used tabs instead of spaces. If it's mandatory in the head you'll need to do something like:

```ruby
stable do
  url "https://taskwarrior.org/download/task-2.4.4.tar.gz"
  sha256 "7ff406414e0be480f91981831507ac255297aab33d8246f98dbfd2b1b2df8e3b"

  depends_on "gnutls" => :optional
end

head do
  url "https://git.tasktools.org/scm/tm/task.git", :branch => "2.5.0", :shallow => false

  depends_on "gnutls"
end
```